explorer/src/providers/cache.tsx

```diff
@@ -9,6 +9,7 @@ export enum FetchStatus {
 export type CacheEntry<T> = {
   status: FetchStatus;
   data?: T;
+  attempts: number;
```

Comment:
Continuing from the [previous discussion](https://github.com/solana-labs/solana/pull/11747#discussion_r476645161)...

> Attempts in this case is just the number of successful fetches completed, not so much the number of confirmation attempts. Maybe attempts is the wrong name.

Hmm yeah attempts means something might not have been successful to me.

> Should this go elsewhere?
Yeah, I don't think we need to track how many successful fetches completed for all cached data. Since the refresh logic is in the component, I think the retries state should be there too. Also, since we never expect confirmations to decrease, I think the right check is to track how many times we get a response of 0 confirmations and bail if it crosses the threshold. Otherwise clear the counter.

Reply:
I agree. This should live in the state.